src/GlobalScreen/Scope/Layout/MetaContent/MetaContent.php

```diff
@@ -116,7 +143,15 @@ public function getCss(): CssCollection
         return $this->css;
     }
 
-    public function getMetaData(): MetaDataCollection
+    public function getOpenGraphMetaData(): ?OpenGraph\TagCollection
```

Comment:
Do I understand it correct: OpenGraph will always be a TagCollection to avoid adding "normal" `MetaData`s here?

Reply:
Correct, we need to distinguish OpenGraph metadata from common metadata at this level because the OpenGraph metadata would be invalid if it was added more than once. The `OpenGraph\TagCollection` was introduced to prevent `Html\TagCollection` from being added there.